ci/ciimage/ubuntu-rolling/install.sh

```diff
@@ -50,6 +51,15 @@ dub_fetch dubtestproject
 dub build dubtestproject:test1 --compiler=ldc2
 dub build dubtestproject:test2 --compiler=ldc2
 
+# Remove debian version of Rust and install latest with rustup.
+# This is needed to get the cross toolchain as well.
+apt-get -y remove rustc || true
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup-init.sh
+chmod +x rustup-init.sh
+./rustup-init.sh -y
```

Comment:
Might be worth doing something like: https://git.sr.ht/~tristan957/dotfiles/tree/master/item/bootstrap/shared/rustup.sh#L2

Reply:
Can be simplified with `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y` indeed.